src/common/RefCountedObj.h

```diff
@@ -74,12 +74,7 @@ class RefCountedObject {
 private:
   void _get() const;
 
-#if defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
-  // crimson is single threaded at the moment
-  mutable uint64_t nref{1};
-#else
   mutable std::atomic<uint64_t> nref{1};
```

Comment:
How broad the change is? From the comments I understand there are 2 users hit by the lack of the atomic. I'm curious how much we're sacrificing to rectify them.

Reply:
it maybe happened when you transfer conn or msg from one shard to another shard by value copy.